protecode/scanning_util.py

```diff
@@ -132,6 +134,75 @@ def _attribute_specs(cls):
         )
 
 
+class Shared:
```

Comment:
stylistic nit: instead of defining a class w/ exactly one static method, why not just define a function?

Reply:
valid comment, was not sure if there will remain only one function. Might be a bit more emphasizing that this is shared between the other classes.